twentytwenty-blocks/experimental-theme.json

```diff
@@ -0,0 +1,21 @@
+{
+	"color": {
+		"primary": "#cd2653",
+		"background": "#f5efe0",
+		"text": "#000"
+	},
+	"typography": {
```

Comment:
I've pushed some changes to https://github.com/WordPress/gutenberg/pull/20530 and these are unnecesary now (they're calculated): `font-size-heading-*`, `line-height-heading`, `font-weight-heading`.